### DISCUSSION TITLE
Permission to use robots.txt tmLanguage

### DISCUSSION THREAD

**Discussion by YashTotale** [Nixinova/NovaGrammars#3] [Questions] [answered]
Hi @Nixinova,

I was wondering if I could use your `robots-txt.tmLanguage.yaml` configuration in a [VS Code extension](https://github.com/hack4impact/vscode-extension) that I'm working on. I can provide appropriate recognition in the documentation of the extension. 

Thanks!

**Comment by Nixinova** [comment, marked as answer]
This repo is licensed under ISC ([file](https://github.com/Nixinova/tmLanguage/blob/main/license.md)), which is equivalent to an MIT license, so feel free to use these files provided credit is given.
